### ISSUE TITLE
Does not work with ESR 45.2

### ISSUE THREAD

**Issue by ParoTheParrot** [gomita/firegestures#135]
Does not work with ESR 45.2

**Comment by dregad** [comment]
Works for me (ESR 45.2.0, FireGestures 1.10.11)

**Comment by reallyuniquename** [comment]
@dregad 
Does "Save Image Now" feature work?

**Comment by dregad** [comment]
> Does "Save Image Now" feature work?

Yes.

**Comment by reallyuniquename** [comment]
@dregad 
And you don't have "Save as..." dialog popping up, right?

**Comment by dregad** [comment]
I do.

**Comment by reallyuniquename** [comment]
Well there should be no "save as" dialog. Otherwise it's no defferent from default "save image as" function/gesture.

**Comment by dregad** [comment]
Right. I'm at work now, and don't have a non-ESR version available for testing. Will check later from home to see if behavior differs.

**Comment by dregad** [comment]
I confirm that from home (Firefox 48.0), the image is saved directly, without dialog

**Comment by reallyuniquename** [comment]
Okay, I opened new issue #141.
